src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+
+  const resultingObject = {};
```

Comment:
The variable name `resultingObject` could be more descriptive. Consider naming it to reflect its purpose, such as `styleObject` or `cssProperties`, to improve code readability.